applications/Flybloc.md

```diff
@@ -0,0 +1,331 @@
+
+## Project Details
+
+### Web3 Strategy
+VeChain, with its dual-token system, offers an optimized balance between transaction costs and speed, making it an attractive solution for applications that demand swift and economical operations. By utilizing VeChain's low-carbon platform, FlyBlock can demonstrate a commitment to eco-friendly practices in our system operations. Furthermore, its established track record in supply chain management and product traceability offers robust tools and frameworks, which can be leveraged for transparent reward tracking and verification in our gaming platform. 
+
+VeChain's capability to handle a high volume of microtransactions seamlessly and expediently would ensure that our users experience instantaneous blockchain interactions, whether it's earning rewards or redeeming them for brand-sponsored offers. As the product scales, we intend to leverage MTT for concurrent transaction handling.
+
+During our tokenomics provenance period, fee delegation will be explored to offset transaction costs as compared to a chargeback to our partners. While our initial MVP is primarily exploring custodial wallets, Sync2 is a strong candidate for self-custody as we explore the most frictionless experience.
+
+Finally, VeChain's compatibility with standard programming languages facilitates easier integration with our existing tech stack. This is especially true given our preference for javascript-based middleware and services, and the use of Connex is already being researched and tested.
+
+## Architecture
+Leveraging Unity3D as our cornerstone, the design encapsulates a primary UI/UX app interfacing with a mobile web full-stack. This ensures a smooth transition and interaction with various game modules. Integration with the Supabase API forms the backbone of our real-time score tracking and user data analytics. The games, though diverse, share common attributes of being engaging and easy to grasp. On the blockchain front, the platform is crafted to reward users for their interactions, with tokens that can be converted into tangible, brand-centric benefits. To top it off, our approach is built on simplicity, ensuring users engage with the brand and its rewards without being entangled in the intricacies of blockchain mechanics.
+
+Please include the team's long-term project plans, operation plans and intentions.
+
+These are the deliverables we submitted for the Harvard 2024 VeChain Hackathon. 
+
+- [Project’s one-pager](https://drive.google.com/file/d/1-XyiktDbbNv989igzI_MBvY1fkbqWTZf/view?usp=drive_link) (provided by BCG at the Hackathon)

```

Comment:
I have requested document access for the account `tony.li@vechain.org`, please kindly check your inbox and give read access to that account.